libs/platform/src/lib/components/table/table.component.ts

```diff
@@ -1435,7 +1440,7 @@ export class TableComponent<T = any> extends Table implements AfterViewInit, OnD
 
     /** @hidden */
     private _getSelectableRows(): TableRow[] {
-        return this._tableRows.filter(({ type }) => type === 'item' || type === 'tree');
+        return this._tableRows.filter(({ type, navigatable }) => (type === 'item' || type === 'tree') && navigatable);
```

Comment:
I would move navigatable as a first condition, since it would be faster with one boolean value to check rather that 2 string values, and then 1 boolean